charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: apiserver-connectivity-check
+  namespace: kube-system
+  labels:
+    garden.sapcloud.io/role: system-component
+    component: apiserver-connectivity-check
+    addonmanager.kubernetes.io/mode: Reconcile
+    origin: gardener
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
+  selector:
+    matchLabels:
+      component: apiserver-connectivity-check
+  template:
+    metadata:
+      annotations:
+        checksum/configmap-gardener-controller-manager-config: {{ include (print $.Template.BasePath "/config.yaml") . | sha256sum }}
+      labels:
+        origin: gardener
+        garden.sapcloud.io/role: system-component
+        component: apiserver-connectivity-check
+    spec:
+      containers:
+      - name: blackbox-exporter
+        image: {{ index .Values.images "blackbox-exporter" }}
+        args:
+        - --config.file=/etc/blackbox_exporter/blackbox.yaml
+        imagePullPolicy: IfNotPresent
+        resources:
+          requests:
+            cpu: 5m
+            memory: 16Mi
+          limits:
+            cpu: 50m
+            memory: 128Mi
```

Comment:
Even more? @mvladev is this really needed? Can't we reduce it to an absolute minimum?

Reply:
```bash
$ kubectl get podmetrics kube-apiserver-7d84bc6fd6-5sjh8 -o yaml
```

```yaml
apiVersion: metrics.k8s.io/v1beta1
containers:
- name: vpn-seed
  usage:
    cpu: 311463n
    memory: 8836Ki
- name: blackbox-exporter
  usage:
    cpu: 252889n
    memory: 19784Ki
- name: kube-apiserver
  usage:
    cpu: 68404140n
    memory: 899872Ki
kind: PodMetrics
metadata:
  creationTimestamp: 2019-01-11T11:46:35Z
  name: kube-apiserver-7d84bc6fd6-5sjh8
timestamp: 2019-01-11T11:45:24Z
window: 30s
```

So we are looking at about `<1m` for `cpu` and about `20Mi` for `memory`. I would say `5m` -> `10m` is okay for CPU and `5Mi` to `35Mi` for memory.

Reply:
Changed requests to `cpu: 5m` `memory: 5Mi` and limits to `cpu: 10m` `memory: 35Mi`